src/pages/Build/TermsOfService.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+	<www-page>
+		<developer-secondary-menu slot="secondary" />
+		<div class="row page-content">
+			<div class="small-12 columns">
+				<h1>Terms Of Service</h1>
+				<p>Last updated: February 19, 2013</p>
+				<h1 style="text-align:center">KIVA DEVELOPER TERMS OF SERVICE</h1>
```

Comment:
@go-bears This is inline style is causing the build fail... you "could" fix it by adding a semi-colon after center; BUT you should just remove it.

Please use class="text-right" instead of a style tag. We don't do inline css.

Reply:
thanks for recommendation! changed 😄